pool.go

```diff
@@ -232,12 +233,20 @@ func (pool *Pool) stopSupervisor() {
 // adds it to the Pool.
 func (pool *Pool) Add(f func(...interface{}) interface{}, args ...interface{}) {
 	job := &Job{f, args, nil, nil, make(chan bool), 0, pool.Next_job_id}
```

Comment:
Did you forget to replace pool.Next_job_id with pool.getNextJobId() here?
This also shows that you're testing the wrong thing in TestGetNextJobId(). We care about actual jobs having unique IDs, not sync/atomic doing what it's supposed to do.

Reply:
yes,i forget..  now what should i do? close this pr?

Reply:
No, just push more commits to your branch and they will appear here.

Reply:
i pushed a commit a moment ago